pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractTopic.java

```diff
@@ -1116,6 +1116,12 @@ public boolean isBrokerPublishRateExceeded() {
         return  getBrokerPublishRateLimiter().isPublishRateExceeded();
     }
 
+    public boolean isBrokerPublishRateExceeded(int numberMessages, int bytes) {
+        // whether broker publish rate exceed
+        return preciseTopicPublishRateLimitingEnable
```

Comment:
It's better two use a distinguish configuration instead of `preciseTopicPublishRateLimitingEnable`

Reply:
fixed.
add config preciseBrokerPublishRateLimiterEnable